player/public/js/content-renderer.js

```diff
@@ -25,7 +25,18 @@ org.ekstep.contentrenderer.loadDefaultPlugins = function(cb) {
     org.ekstep.contentrenderer.initPlugins('', 'coreplugins');
     var globalConfig = EkstepRendererAPI.getGlobalConfig();
     globalConfig.isCorePluginsPackaged && jQuery("body").append($("<script type='text/javascript' src='./coreplugins.js?" + globalConfig.version + "'>"));
-    org.ekstep.contentrenderer.loadPlugins(globalConfig.defaultPlugins, [], function() {
+    var plugins = globalConfig.defaultPlugins;
```

Comment:
Dont move this logic here. Keep it in launcher plugin only(org.ekstep.launcher), but call the method through event.

Presently is it calling loadCommonPlugins() method in the initialize method of it. Instead we are calling through event. So launcher plugins functionality will be still exist in org.ekstep.launcher plugin only.